src/main/java/net/mcreator/ui/minecraft/TextureHolder.java

```diff
@@ -154,8 +154,10 @@ public void setActionListener(ActionListener actionListener) {
 
 				@Override public synchronized void paintIcon(Component c, Graphics g, int x, int y) {

```

Comment:
I know this is not your fault (I wrote this code), but I just realized we have a potential memory leak here (https://stackoverflow.com/questions/68864382/swing-custom-painting-should-graphic-object-be-disposed, https://stackoverflow.com/questions/42875585/disposing-graphics2d-objects-created-by-graphics2d-create) as g2 is not disposed.

Could you add this to this PR as we are changing the rendering code here?

Reply:
✔️ Fixed